app/views/events/settings/_tags.html.erb

```diff
@@ -4,13 +4,24 @@
   <h3 class="mb1" id="cards_heading">Create a new tag</h3>
   <div class="card mb3">
     <%= form_with(url: event_tags_path(@event)) do |form| %>
-      <div class="field">
-        <%= form.label :label, "Tag name", class: "mb-1" %>
-        <%= form.text_field :label, placeholder: ["🧃 Food & drinks", "🧋 Bubble tea", "🚕 Transportation", "🛬 Flights", "🔨 Supplies"].sample, disabled: %>
+      <div class="flex gap-5">
+        <% placeholder = ["🧃 Food & drinks", "🧋 Bubble tea", "🚕 Transportation", "🛬 Flights", "🔨 Supplies"].sample %>
+        <div class="field">
+          <div data-controller="emoji-picker" class="relative" data-emoji-picker-target="container">
+            <%= form.label :emoji, "Emoji", class: "mb-1" %>
+            <%= form.text_field :emoji, placeholder: placeholder[0], class: "!w-10", disabled:, data: { "emoji-picker-target": "input", "action": "focus->emoji-picker#togglePicker" } %>
+            <emoji-picker data-emoji-picker-target="picker" data-action="emoji-click->emoji-picker#addEmoji" class="fixed mt-3 z-[300]" style="display: none;"></emoji-picker>
+          </div>
+        </div>
+
+        <div class="field">
+          <%= form.label :label, "Name", class: "mb-1" %>
+          <%= form.text_field :label, placeholder: placeholder[1..], disabled: %>
```

Comment:
Substringing for the first character unfortunately won't work for multicharcter emojis such as 🧑‍🚀 for skin-toned emojis. I'd suggest refactoring `placeholder` to be a hash with the keys `emojis` and `label`.